### DISCUSSION TITLE
百度网盘 无法在线播放 开启代理也不行

### DISCUSSION THREAD

**Discussion by ddaodan** [AlistGo/alist#1940] [Q&A] [answered]
https://pan.ddaodan.cn/Baidu/%E6%88%91%E7%9A%84%E8%B5%84%E6%BA%90/VTuber%20Fes%20Japan%202022%20DAY2.mp4

无论在后台是否开启代理，设置非官方接口，都不能播放，但是在之前v2是可以播放的

**Comment by ddaodan** [comment, marked as answer]
啊这，自己好了。。。
好像是重启之后就好了

**Reply by KaDoO777** [reply]
兄弟，你好像没有设置权限，Alist的所有文件都能被看到。。。

**Comment by ddaodan** [comment]
昨天刚升级的还没弄好发件人: ***@***.*** ***@***.***> 代表 KaDoO777 ***@***.***>发送时间: 2022年10月10日星期一 12:44收件人: alist ***@***.***>抄送: 731882332 ***@***.***>; author ***@***.***>主题: Re: [alist-org/alist] 百度网盘 无法在线播放 开启代理也不行 (Discussion #1940)

兄弟，你好像没有设置权限，Alist的所有文件都能被看到。。。
